src/main/webapp/scripts/analyze.js

```diff
@@ -1,6 +1,6 @@
 async function analyzeVideo() {
   const videoId = document.getElementById('video-url').value.split('v=')[1];
```

Comment:
Nit: Could this be changed into a regex to capture the videoId? Users might be in some weird version of the URL with more stuff after the video id.
https://stackoverflow.com/a/8260383

Reply:
Changed. Wow there were URL's I haven't even heard of before, what in the world is youtube-nocookie?